### DISCUSSION TITLE
TypeScript Warning with ChartTooltipProps

### DISCUSSION THREAD

**Discussion by blentz100** [mantinedev/mantine#5807] [Q&A]
When I use

```
interface ChartTooltipProps {
  label: string;
  payload: Record<string, any>[] | undefined;
}
```

(full file is [here](https://github.com/blentz100/brendanlentz.com/blob/main/components/HabitLineChart/HabitLineChart.tsx))

as documented here: https://mantine.dev/charts/line-chart/#custom-tooltip

I get a TypeScript warning when I build:

_24:27  Warning: Unexpected any. Specify a different type.  @typescript-eslint/no-explicit-any_

Is that just how Records are supposed to work? Or should that `any` be a specific type?

**Comment by rtivital** [comment]
What line of code causes the issue?

**Reply by blentz100** [reply]
@rtivital 
This is the line:

`payload: Record<string, any>[] | undefined;`

**Reply by rtivital** [reply]
If your project does not allow any, you can specify different type
